### PR TITLE
XRDDEV-1584 UI fixes

### PR DESCRIPTION
**Pull request by Riippi**
https://jira.niis.org/browse/XRDDEV-1584
https://jira.niis.org/browse/XRDDEV-1585
https://jira.niis.org/browse/XRDDEV-1586

Multiple UI changes of varying size.

### CODE REVIEW

File: src/proxy-ui-api/frontend/src/components/layout/AppDropMenu.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    <v-menu bottom right>
+      <template v-slot:activator="{ on }">
+        <v-btn text v-on="on" class="no-uppercase">
+          {{ username }}
+          <v-icon>mdi-chevron-down</v-icon>
+        </v-btn>
+      </template>
+
+      <v-list>
+        <v-list-item id="logout-list-tile" @click="logout">
```
Comment (TJaakkola) [inline src/proxy-ui-api/frontend/src/components/layout/AppDropMenu.vue:37]: I think this is missing data-test attribute

File: src/proxy-ui-api/frontend/src/components/ui/Snackbar.vue [hunk 2]
```diff
@@ -26,38 +26,40 @@
...
+        :timeout="notification.timeout"
+        v-model="notification.show"
+        :color="colors.Success10"
+        multi-line
+        class="success-snackbar"
+        :min-width="760"
+        @input="closeSuccess(notification.timeAdded)"
+      >
+        <div class="row-wrapper-top scrollable identifier-wrap">
+          <v-icon color="#0CC177">icon-Checker</v-icon>
```
Comment (TJaakkola) [inline src/proxy-ui-api/frontend/src/components/ui/Snackbar.vue:43]: Color code here is a magicvalue. Prefer using a variable.

**Review by TJaakkola — CHANGES_REQUESTED** [review]
Some minor things noticed.

File: src/proxy-ui-api/frontend/src/components/layout/AppDropMenu.vue (hunk 1 above)
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/components/layout/AppDropMenu.vue:37]: Fixed

File: src/proxy-ui-api/frontend/src/components/layout/AppIcon.vue [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    home(): void {
+      this.$router
+        .replace({
+          name: this.$store.getters.firstAllowedTab.to.name,
+        })
+        .catch((err) => {
+          // Ignore the error regarding navigating to the same path
+          if (err.name === 'NavigationDuplicated') {
+            // eslint-disable-next-line no-console
+            console.log('Duplicate navigation');
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/components/layout/AppIcon.vue:52]: Changed to info.

File: src/proxy-ui-api/frontend/src/components/ui/Snackbar.vue (hunk 2 above)
Comment (Riippi) [reply src/proxy-ui-api/frontend/src/components/ui/Snackbar.vue:43]: Fixed

File: src/shared-ui/src/components/XrdSearch.vue [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+ -->
+<template>
+  <div>
+    <v-icon @click="closed = false" v-if="closed" class="icon-closed"
+      >mdi-magnify</v-icon
+    >
+    <v-text-field
+      v-if="!closed"
+      :label="label"
+      data-test="search-clients-input"
```
[the remark this answers is not in the document]
Comment (Riippi) [reply src/shared-ui/src/components/XrdSearch.vue:35]: Fixed

File: src/proxy-ui-api/frontend/src/components/layout/AppDropMenu.vue [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+ -->
+<template>
+  <div class="drop-menu">
+    <v-menu bottom right>
+      <template v-slot:activator="{ on }">
+        <v-btn text v-on="on" class="no-uppercase">
```
Comment (TJaakkola) [inline src/proxy-ui-api/frontend/src/components/layout/AppDropMenu.vue:30]: Just now noticed this is also missing data-test attribute

**Review by TJaakkola — APPROVED** [review]

**Pull request merged**